components/ui/tabcounter/src/main/java/mozilla/components/ui/tabcounter/TabCounter.kt

```diff
@@ -60,32 +57,35 @@ open class TabCounter @JvmOverloads constructor(
         animationSet = createAnimatorSet()
     }
 
-    fun getText(): CharSequence {
-        return text.text
+    /**
```

Comment:
Don't think we need to change this

Reply:
But we compare strings in test. They will fail, right?

Reply:
Ah I see. You can just call `toString()` in the test.  Changing this changes the public API of the class which might have unforeseen consequences.

Reply:
There was too much but okey, gonna change them.
Should I change TabCounterToolbarButtonTest.kt because i didn't touch them.

![resim](https://user-images.githubusercontent.com/17825767/85472603-bdbc4c00-b5ba-11ea-94e6-af4aba3bc912.png)

Reply:
That's weird.  Why didn't we have this issue before this?  Is there something we're missing?

Reply:
Wait, its working fine without `.toString()`.
Does kotlin compare CharSequence and string?

Reply:
I've pushed commit, wait and see :)

Reply:
It's working fine.